runtime/opt/taupage/init.d/10-prepare-disks.py

```diff
@@ -57,13 +57,18 @@ def attach_volume(ec2, volume_id, attach_as):
         sys.exit(3)
 
 
-def wait_for_device(device, max_tries=3, wait_time=5):
+def wait_for_device(device, max_tries=12, wait_time=5):
     """Gives device some time to be available in case it was recently attached"""
     tries = 0
-    while tries < max_tries and not os.path.exists(device):
+    while tries < max_tries:
+        if os.path.exists(device):
+            return
         logging.info("Waiting for %s to stabilize..", device)
         tries += 1
         sleep(wait_time)
+    logging.error("Failed to wait for %s device to become available after %s seconds",
+                  device, max_tries * wait_time)
+    sys.exit(2)
```

Comment:
The reason is: https://github.com/zalando/planb-cassandra/issues/93#issuecomment-263242986
> The problem seems to be that Taupage's wait_for_device by default only waits for up to 15 seconds. If you are unlucky to attach the volume just after mount failure, the next step is to format it.

Basically, it is safer to stop the boot up procedure than risk formatting some really big EBS with some really important data ;-)